cypress/support/commands.js

```diff
@@ -39,3 +43,51 @@ Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', pas
     password
   });
 });
+
+Cypress.Commands.add('login', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/users/login', {
+      user: {
+        email,
+        username,
+        password,
+      },
+    }).then((response) => {
+      const user = {
+        bio: response.body.user.bio,
+        effectiveImage:
+          'https://static.productionready.io/images/smiley-cyrus.jpg',
+        email: response.body.user.email,
+        image: response.body.user.image,
+        token: response.body.user.token,
+        username: response.body.user.username,
+        id: response.body.user.id
+      };
+      window.localStorage.setItem('user', JSON.stringify(user));
+      cy.setCookie('drash_sess', response.body.user.token);
+    });
+  }
+);
+
+Cypress.Commands.add('createArticle', (title, description, body) => {
+  cy.getCookie('drash_sess').then((token) => {
+    const authToken = token.value;
+    const author_id = response.body.user.id;
+
+    cy.request({
+      method: 'POST',
+      url: '/articles',
+      body: {
+        article: {
+          title,
+          description,
+          body,
+          tagList: [],
+          author_id,
+        },
+      },
+      headers: {
+        Cookie: `drash_sess=${authToken}`,
```

Comment:
It is not a good practice to set cookies manually, it is better to let the server set it via HTTP responses. Consider using cy.setCookie instead.